src/audio/drc/drc_math_lib/db2mag/c_fixpt/drc_db2mag_types.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * File: drc_db2mag_types.h
+ *
+ * Author :Shriram Shastry
+ * Copyright (c) 2016, Intel Corporation All rights reserved.
+ */
+
+#ifndef DRC_DB2MAG_TYPES_H
+#define DRC_DB2MAG_TYPES_H
+
+/* Include Files */
+#include "typedef.h"
+
+/* Type Definitions */
+#ifndef typedef_struct0_T
+#define typedef_struct0_T
+
+typedef struct
+{
+  uint8_T u1;
+  uint8_T u2;
+}
+struct0_T;
```

Comment:
A very non-descriptive struct name. Also we don't use "typedef structs" in normal components, only some xtensa HAL codes contains such.